src/pages/index.js

```diff
@@ -90,3 +172,38 @@ editButton.addEventListener('click', ()=>{
 addCardButton.addEventListener('click', ()=>{
   cardModal.open()
 });
+
+avatar.addEventListener('mouseover', ()=>{
+  avatarOverlay.classList.add('profile__photo-overlay_visible')
+})
+
+avatarOverlay.addEventListener('mouseout', ()=> {
+  avatarOverlay.classList.remove('profile__photo-overlay_visible')
+})
+
+avatarOverlay.addEventListener('click', ()=>{
+  changeAvatarModal.open()
+})
+
+//On window load
+api.getCurrentUser().then(data => {
+  userInfo.setUserInfo(data)
+});
+
+api.checkAllData(()=>{
```

Comment:
`checkAllData` has everything you need in the response. Please, don't make a lot of redundant requests to the server

You need to combine the requests of the user's info and the cards together in `Promise.all` to achieve that

```js

api.checkAllData()
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```